### ISSUE TITLE
Create VM fail on ESXi 5.5 standalone

### ISSUE THREAD

**Issue by tommymunter** [terraform-providers/terraform-provider-vsphere#265]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform
issue trackers reserved for bug reports and feature requests. For general usage
questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.11.0
+ provider.vsphere v1.0.1

Terraform v0.11.1
+ provider.vsphere v1.0.1

Terraform v0.10.8
* provider.vsphere: version = "~> 1.0"

no difference in all 3 versions.

### Affected Resource(s)
- `vsphere_virtual_machine`

### Terraform Configuration Files

```hcl
data "vsphere_datacenter" "dc" {
  name = "ha-datacenter"
}

data "vsphere_datastore" "datastore" {
  name          = "datastore1"
  datacenter_id = "${data.vsphere_datacenter.dc.id}"
}

data "vsphere_resource_pool" "pool" {}

data "vsphere_network" "mgmt_lan" {
  name          = "Yellow network"
  datacenter_id = "${data.vsphere_datacenter.dc.id}"
}

resource "vsphere_virtual_machine" "test2" {
  name             = "server-40"
  resource_pool_id = "${data.vsphere_resource_pool.pool.id}"
  datastore_id     = "${data.vsphere_datastore.datastore.id}"

  num_cpus   = 1
  memory     = 2048
  wait_for_guest_net_timeout = 0

  network_interface {
   network_id     = "${data.vsphere_network.mgmt_lan.id}"
   adapter_type   = "vmxnet3"
   use_static_mac = true
   mac_address    = "00:50:56:14:c2:40"
  }

  disk {
   size             = 16
   name             = "server-40.vmdk"
   disk_sharing     = "sharingNone"
   eagerly_scrub    = false
   thin_provisioned = true
  }
}
```

### Expected Behavior

create new vm

### Actual Behavior

Error: Error applying plan:

1 error(s) occurred:

* vsphere_virtual_machine.test2: 1 error(s) occurred:

* vsphere_virtual_machine.test2: error creating virtual machine: ServerFaultCode:
Unexpected element tag "sharing" seen

while parsing serialized DataObject of type vim.vm.device.VirtualDisk.FlatVer2BackingInfo
at line 2, column 1537

while parsing property "backing" of static type VirtualDeviceBackingInfo

while parsing serialized DataObject of type vim.vm.device.VirtualDisk
at line 2, column 1476

while parsing property "device" of static type VirtualDevice

while parsing serialized DataObject of type vim.vm.device.VirtualDeviceSpec
at line 2, column 1280

while parsing property "deviceChange" of static type ArrayOfVirtualDeviceConfigSpec

while parsing serialized DataObject of type vim.vm.ConfigSpec
at line 2, column 157

while parsing call information for method CreateVM_Task
at line 2, column 83

while parsing SOAP body
at line 2, column 77

while parsing SOAP envelope
at line 2, column 0

while parsing HTTP request for method createVm
on object of type vim.Folder
at line 1, column 0

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
Standalone
ESXi 5.5.0 Update 3

### References

**Comment by vancluever** [comment]
Hey @tommymunter - sorry this is happening for you. This should have been corrected as disk sharing is not available on vSphere 5.5 - we will try to reproduce on 5.5 as soon as we can!

**Comment by vancluever** [comment]
Hey @tommymunter just confirmed this is indeed a bug and I'm working on a fix!

**Comment by d10sbt** [comment]
we have the same problem ... any workaround to be able to continue working?

**Comment by tommymunter** [comment]
Hi for #265 there is à fix in 1.0.2
I am running the latest version now. Works fine

**Comment by d10sbt** [comment]
ok @tommymunter ! I've tried it and it works perfectly 
thanks!!!
